streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamThreadTest.java

```diff
@@ -2552,7 +2552,8 @@ public Set<TopicPartition> partitions() {
     }
 
     @ParameterizedTest
-    @MethodSource("data")        
+    @MethodSource("data")
+    @SuppressWarnings("deprecation")
```

Comment:
This test verifies that the handler is doing the right thing. Thus, we should update the test to use the new config and not add this annotation